src/pages/Board/index.tsx

```diff
@@ -32,30 +48,46 @@ const Board = () => {
             작성하기
           </S.WriterButton>
         </S.WriteContainer>
-        {notifications.map((data, i) => (
-          <S.BoardBoxContainer
-            key={i}
-            onClick={() => {
-              router(`/boards/${i}`);
-            }}
-          >
+        {projectDetail?.map((data, i) => (
+          <S.BoardBoxContainer key={i}>
             <S.ProfileInfoContainer>
-              <div>{data.project}</div>
-              <div>{data.user}</div>
+              <S.Title
+                onClick={() => {
+                  router(`/boards/${i}`);
+                }}
```

Comment:
```suggestion
                onClick={() => {
                // 이것보단
                  router(`/boards/${i}`);
                 handleGoBoardDetailPage(); // 이게 명시적이고 읽기 쉽겠져?
                }}
```

상황에 따른 함수 추상화를 통해 비즈니스 로직을 알려준다면 개발자 경험에 좋을것같아요

Reply:
넵 감사합니다